src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -208,9 +248,14 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
 
         try {
             ResponseContentSupplier responseContentSupplier = new ResponseContentSupplier(response);
-            logResponse(build, logger, responseContentSupplier);
+            logResponse(workspace, logger, responseContentSupplier);
```

Comment:
The workspace is required only when the user specifies an output file. I was planning on dealing with that in "phase 2", but okay, I can deal with it in this pull-request.